index.js

```diff
@@ -521,6 +544,11 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this.valueOf() <= other.valueOf();
+  }
+
```

Comment:
I understand the philosophy behind this thinking, but I don't see the immediate utility. Why might I want to distinguish between the boxed vs. primitive version of a value?